scripts/ci.sh

```diff
@@ -13,12 +13,19 @@ if [ "$TEST" = "unit" ]; then
   npm run test:unit
   npm run dtslint
 
-elif [ "$TEST" = "unit_and_e2e" ]; then
+elif [ "$TEST" = "unit_and_e2e_clients" ]; then
 
   npm run build
-  npm run test:e2e:all
+  npm run test:e2e:ganache
+  npm run test:e2e:geth:insta
+  npm run test:e2e:geth:auto
   npm run test:unit
-  npm run dtslint
```

Comment:
This is already being run in 3 other containers and takes a while, without adding to coverage measurements. Is there a reason to keep in the Node 10 container specifically?

Reply:
One possibility is to move building and linting into their own CI job in a Node 10 container, if they're effectively duplicate elsewhere.

Reply:
> One possibility is to move building and linting into their own CI job in a Node 10 container, if they're effectively duplicate elsewhere.

Yes, this sounds like a good idea 👍

Reply:
Made this change in 6d3ef0c6df5900f3a8e3bbdd2ad719fa90ed133d